plugins/extractors/postgres/README.md

```diff
@@ -9,6 +9,8 @@ source:
     host: localhost:5432
     user_id: admin
```

Comment:
any reason why we don't make postgres to use connection url like the others?

Reply:
currently postgres extractor requires `host` and `database` value to be used as part of the `URN`.

If we are using connection URL, we would have to get the host and database value from the connection string.
It is totally possible but there could be different formats of connection string and we might need to handle each formats if we want to extract the host and database.

We could:
- change postgres to use connection string and handle connection string formats
- change postgres to use connection string and change how we generate urn so that it does not depend on host

Reply:
If that's the case, sooner or later the other DB also will have `host` as part of its URN right? or won't it happen?

Reply:
If later other DB extractor requires `host` as a part of `URN` then either we have to change 
- how we generate `urn` to satisfy all connection string format 
- have to use username and password separately just like we use currently
wdyt ??

Reply:
as for now, I don't think that will be much issue. Just need to take a note that other DBs are still using URN `database_name.table_name`. The URN afaik is not final and could still be evolving.

In that case, can you please also update the resource URN in the output in this Postgres [README](https://github.com/odpf/meteor/blob/main/plugins/extractors/postgres/README.md)?

Reply:
Even I think we should standardize as @mabdh pointed out, use the connection URL here as well. It's pretty easy to parse connection strings as the logic of parsing can stay in individual extractors right?

Reply:
Yes better to make it consistent so it is predictable. 

It is not that difficult to parse URI, we can use [net/url](https://pkg.go.dev/net/url#Parse) or [libpq](https://github.com/lib/pq/blob/master/url.go#L32).